SPECS/xapi.spec

```diff
@@ -77,6 +77,7 @@ Patch1006: xapi-24.19.2-update-new-fingerprint-fields-on-DB-upgrade.backport.pat
 Patch1007: xapi-24.19.2-more-fingerprint-field-updates-fixes.XCP-ng.patch
 Patch1008: xapi-24.19.2-ipv6-reset-networking.XCP-ng.patch
 Patch1009: xapi-24.19.2-keep-address-type-network-reset.XCP-ng.patch
+Patch1010: xapi-24.19.2-keep-ipv6-management-disable.XCP-ng.patch
```

Comment:
Since it's already merged, you could add a comment stating which future version of XAPI is supposed to embed it.

Reply:
I'll do it in next PR since everything have the been merged for the same version :+1: